app/elements/pages/page-browse.html

```diff
@@ -241,7 +241,6 @@ <h2 class="flex">[[pageTitle]]</h2>
         }
 
         this.fire('page-meta', { title: this.pageTitle });
-        this.$.drawerPanel.closeDrawer();
```

Comment:
Do you think it would it be ok to just check if the query is defined?
Maybe also check if the query is an 'element-name' & close the drawer.
```
if (!this.q) {
    this.$.drawerPanel.closeDrawer();
}
```